packages/client/package.json

```diff
@@ -12,6 +12,7 @@
     "react-router-dom": "^5.1.2",
     "react-signature-canvas": "^1.0.3",
     "styled-components": "^5.0.1",
+    "styled-react-modal": "^2.0.1",
```

Comment:
How many KB's does this package contribute?

Reply:
Good point, I ran `yarn analyze` inside client package after adding this and its **6.09 KB** in the finally built chunk, which is good in my opinion 

<img width="685" alt="Screenshot 2020-05-30 at 9 47 42 PM" src="https://user-images.githubusercontent.com/53977855/83337933-b22d7c00-a2bf-11ea-9cd9-6a8e447232a9.png">

Reply:
Thanks!